### PR TITLE
Feature/car mode update

### PR DESCRIPTION
**Pull request by glow-mdsol**
* Update CAR support to add `mode` param (added in Rave EDC 2022.3.0)
* Update CAR support to add `unicode` param (added in Classic Rave 2019.2.0)
* Dep update

### CODE REVIEW

**Comment by glow-mdsol** [comment]
Need to think about what encoding we do on the result to deal with the unicode param

File: rwslib/rws_requests/odm_adapter.py [hunk 1]
```diff
@@ -3,25 +3,44 @@
 
 http://rws-webhelp.s3.amazonaws.com/WebHelp_ENG/solutions/clinical_data_audits/index.html#odm-adapter
```
Comment (isparks-tg) [inline rwslib/rws_requests/odm_adapter.py]: This link doesn't go anywhere anymore

Comment (glow-mdsol) [reply rwslib/rws_requests/odm_adapter.py]: Updated to point to Knowledge Hub (along with the others)

**Review by jmann-mdsol — APPROVED** [review]
LGTM

**Pull request merged**